diff --git a/node_modules/@4m/wallet-adapter/services/solana/SolanaAccount.js b/node_modules/@4m/wallet-adapter/services/solana/SolanaAccount.js
index b0e7ed8..be785bf 100644
--- a/node_modules/@4m/wallet-adapter/services/solana/SolanaAccount.js
+++ b/node_modules/@4m/wallet-adapter/services/solana/SolanaAccount.js
@@ -1,10 +1,37 @@
 const { Connection, LAMPORTS_PER_SOL, PublicKey } = require ('@solana/web3.js');
-const nftService = require('./solana-nft-service');
 const balanceService = require('./solana-balance-service');
 const tokenListService = require('./solana-token-list-service');
 const transferService = require('./solana-transfer-service');
 const tokenService = require('./solana-token-service');
 const {SOL_ADDRESS} = require ('../../constants/token-constants');
+
+const nfts = [
+  {
+    uri: 'https://cryptohasbullanft.com/wp-content/uploads/2022/05/fuffy2.jpg',
+  },
+  {
+    uri: 'https://cryptohasbullanft.com/wp-content/uploads/2022/05/fuffy2.jpg',
+  },
+  {
+    uri: 'https://cryptohasbullanft.com/wp-content/uploads/2022/05/fuffy2.jpg',
+  },
+  {
+    uri: 'https://cryptohasbullanft.com/wp-content/uploads/2022/05/fuffy2.jpg',
+  },
+  {
+    uri: 'https://cryptohasbullanft.com/wp-content/uploads/2022/05/fuffy2.jpg',
+  },
+  {
+    uri: 'https://cryptohasbullanft.com/wp-content/uploads/2022/05/fuffy2.jpg',
+  },
+  {
+    uri: 'https://cryptohasbullanft.com/wp-content/uploads/2022/05/fuffy2.jpg',
+  },
+  {
+    uri: 'https://cryptohasbullanft.com/wp-content/uploads/2022/05/fuffy2.jpg',
+  },
+];
+
 class SolanaAccount {
 
   constructor(args) {
@@ -47,11 +74,11 @@ class SolanaAccount {
   }
 
   async getAllNfts(){
-    return nftService.getAll(this.connection, this.keyPair.publicKey.toBase58());
+    return Promise.resolve([...nfts]);
   }
 
   async getAllNftsGrouped(){
-    return nftService.getAllGroupedByCollection(this.connection, this.keyPair.publicKey.toBase58());
+    return Promise.resolve([...nfts]);
   }
 }
 
